### PR TITLE
feat(📦): Add contains function

### PR DESCRIPTION
**Pull request by wcandillon**

### CODE REVIEW

File: src/AnimationRunners.ts [hunk 1]
```diff
@@ -12,24 +11,13 @@ const {
...
-export type Clock = Parameters<typeof clockRunning>[0];
-export type Node = ReturnType<typeof add>;
-export type Adaptable<T> = Node | T;
-
 export function runDecay(
-  clock: Clock,
-  value: Adaptable<number>,
-  velocity: Adaptable<number>,
-  rerunDecaying: any,
+  clock: Animated.Clock,
```
Comment (dsznajder) [inline src/AnimationRunners.ts]: Woo, I didn't know that there is this type inside.

File: src/index.ts [hunk 2]
```diff
@@ -24,48 +23,49 @@ const {
...
 ) => interpolate(value, {
   inputRange: [0, 1],
   outputRange: [origin, destination],
 });
 
-export const lookup = (
-  array: Adaptable<number>[],
-  index: Adaptable<number>,
-  notFound: Node = new Value(),
+export const find = (
```
Comment (dsznajder) [inline src/index.ts]: You should probably change README.md also

Comment (wcandillon) [reply src/index.ts]: yes and yes, thank you :)

Comment (wcandillon) [reply src/index.ts]: I will add the breaking change note in the squash commit.

**Pull request merged**